kjourney/src/app/login/login.component.html

```diff
@@ -16,14 +16,23 @@
 							<label for="Password">Password</label>
 					<div><input type="password" [(ngModel)]="newUser.password" class="form-control" name="Password"  required> </div>
 				  </div>
-					<div class = "form-group">
-
-							<button [disabled]="!addForm.valid" class="btn btn-primary">
-									Login
-							</button>
+				  <div class="row">
+					  <div class="col">
+						<div class = "form-group">
 
+								<button [disabled]="!addForm.valid" class="btn btn-primary">
+										Login
+								</button>
+					</div>
+				</div>
+				<div class="col">
+					<div class = "reg-link">
+						<a href="/register-user">Register user</a>
```

Comment:
a consistent format for urls, you are currently using camel case in most of your endpoints so stick to it